### PR TITLE
[INTL] Update Romanian (ro-RO) translation

### PR DESCRIPTION
**Pull request by AndreiMiloiu**
Attendum to https://github.com/reactos/reactos/pull/6517

### CODE REVIEW

**Comment by AndreiMiloiu** [comment]
Cc @reactos/lang-romanian

File: dll/cpl/intl/lang/ro-RO.rc [hunk 1]
```diff
@@ -125,21 +125,18 @@ CAPTION "Oră"
...
     COMBOBOX IDC_TIMEAMSYMBOL, 149, 86, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Sim&bolul P.M.:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "Simbol &PM:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEPMSYMBOL, 149, 104, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "", -1, 7, 140, 230, 80, WS_VISIBLE
-    LTEXT "Notația orei:\n\n\
-h = oră     m = minut    s = secundă\nt = am sau pm\n\
-h = 12 ore    H = 24 ore\n\
-hh, mm, ss = cu zerouri   h, m, s = fără zerouri", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Notarea formatelor de oră\nh = oră     m = minut     s = secundă     t = am sau pm\n\nh = 12 ore\nH = 24 ore\n\nhh, mm, ss = zero iniţial\nh, m, s = fără zero iniţial", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
```
Comment (binarymaster) [inline dll/cpl/intl/lang/ro-RO.rc:139]:
```suggestion
    LTEXT "Notarea formatelor de oră\n\n\
h = oră     m = minut     s = secundă     t = am sau pm\n\n\
h = 12 ore   H = 24 ore\n\
hh, mm, ss = zero iniţial   h, m, s = fără zero iniţial", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
```

File: dll/cpl/intl/lang/ro-RO.rc [hunk 2]
```diff
@@ -125,21 +125,21 @@ CAPTION "Oră"
...
     COMBOBOX IDC_TIMEAMSYMBOL, 149, 86, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Sim&bolul P.M.:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "Simbol &PM:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEPMSYMBOL, 149, 104, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "", -1, 7, 140, 230, 80, WS_VISIBLE
-    LTEXT "Notația orei:\n\n\
-h = oră     m = minut    s = secundă\nt = am sau pm\n\
-h = 12 ore    H = 24 ore\n\
-hh, mm, ss = cu zerouri   h, m, s = fără zerouri", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Notarea formatelor de oră\n\n\
```
Comment (binarymaster) [inline dll/cpl/intl/lang/ro-RO.rc:139]: Keep this layout in sync with en-US, if it doesn't fit visually, tweak the coordinates.

Comment (AndreiMiloiu) [reply dll/cpl/intl/lang/ro-RO.rc:139]: I see now. Thank you @binarymaster!

**Comment by AndreiMiloiu** [comment]
Any news?

File: dll/cpl/intl/lang/ro-RO.rc [hunk 3]
```diff
@@ -14,23 +14,23 @@ CAPTION "Opțiuni regionale"
...
+    LTEXT "Dată lungă:", -1, 16, 146, 68, 10
     EDITTEXT IDC_NUMSAMPLE_EDIT, 89, 86, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_MONEYSAMPLE_EDIT, 89, 101, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_TIMESAMPLE_EDIT, 89, 116, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_SHORTTIMESAMPLE_EDIT, 89, 131, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
     EDITTEXT IDC_FULLTIMESAMPLE_EDIT, 89, 146, 140, 12, ES_READONLY | WS_CHILD | WS_VISIBLE | WS_GROUP | NOT WS_TABSTOP
-    GROUPBOX "Locație geografică", -1, 5, 172, 234, 55
-    LTEXT "Specificați locația astfel încât unele ser&vicii să vă poată furniza informații locale; de exemplu, știri și rapoarte meteo.", -1, 14, 184, 210, 24
+    GROUPBOX "Amplasare", -1, 5, 172, 234, 55
+    LTEXT "P&entru ca serviciile să vă ofere informaţii regionale, cum ar fi ştirile sau vremea, selectaţi locul unde vă aflaţi:", -1, 14, 184, 210, 24
```
Comment (JoachimHenze) [inline dll/cpl/intl/lang/ro-RO.rc:33]: Why did you turn the trailing `.` to a `:` ? That is not what our other languages do.

File: dll/cpl/intl/lang/ro-RO.rc [hunk 4]
```diff
@@ -39,83 +39,83 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
+    GROUPBOX "Limbi pentru programele non-Unicode", -1, 5, 5, 234, 90
     COMBOBOX IDC_LANGUAGE_COMBO, 14, 75, 217, 160, CBS_DROPDOWNLIST | CBS_NOINTEGRALHEIGHT | CBS_AUTOHSCROLL | WS_VSCROLL | WS_TABSTOP | CBS_SORT
-    LTEXT "Această opțiune specifică limba de afișare pentru programele care nu utilizează Unicode în textele din meniuri și ferestrele de dialog. Opțiunea nu afectează aplicațiile Unicode, dar va afecta toți utilizatorii din sistem.", -1, 14, 18, 223, 33
-    LTEXT "Alegeți limba ce corespunde cu limba aplicațiilor ne-Unicode pe care doriți să le utilizați:", -1, 14, 55, 223, 18
-    GROUPBOX "Tabele de conversie cod-pagină", -1, 5, 101, 234, 88
+    LTEXT "Setările acestui sistem permit programelor non-Unicode să afişeze meniuri şi dialoguri în limba lor de origine. Acest fapt nu afectează programele Unicode, dar se aplică pentru toţi utilizatorii acestui computer.", -1, 14, 18, 223, 33
+    LTEXT "&Selectaţi o limbă care să corespundă versiunii de limbă a programelor non-Unicode pe care le veţi utiliza:", -1, 14, 55, 223, 18
+    GROUPBOX "Tabele de &conversie a paginilor de cod", -1, 5, 101, 234, 88
```
Comment (JoachimHenze) [inline dll/cpl/intl/lang/ro-RO.rc:60]: which sense does it make to add an accelerator to a group-box?

File: dll/cpl/intl/lang/ro-RO.rc [hunk 5]
```diff
@@ -39,83 +39,83 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
     CONTROL "", IDC_CONV_TABLES, "SysListView32", LVS_REPORT | LVS_SORTASCENDING | LVS_NOCOLUMNHEADER | WS_BORDER | WS_TABSTOP, 14, 114, 217, 70
-    GROUPBOX "Opțiuni implicite pentru conturi de utilizator", -1, 5, 193, 234, 32
-    CHECKBOX "Apli&că aceste opțiuni atât pentru contul de utilizator curent cât și pentru cel implicit", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
+    GROUPBOX "Setările implicite pentru conturile de utilizator", -1, 5, 193, 234, 32
+    CHECKBOX "Se aplică toate setările la contul utilizatorului curent şi la profilul utilizator &implicit", IDC_APPLY_CUR_USER_DEF_PROFILE, 12, 201, 220, 22, BS_MULTILINE | BS_TOP
```
Comment (JoachimHenze) [inline dll/cpl/intl/lang/ro-RO.rc:63]: Are you sure that romanian 2k3 does use &i here? MS does typically not recommend that. If you can prove it with a pic, then it is fine ofc.

File: dll/cpl/intl/lang/ro-RO.rc [hunk 6]
```diff
@@ -125,21 +125,21 @@ CAPTION "Oră"
...
-    LTEXT "Sim&bolul P.M.:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
+    LTEXT "Simbol &PM:", -1, 13, 106, 130, 10, WS_VISIBLE | WS_GROUP
     COMBOBOX IDC_TIMEPMSYMBOL, 149, 104, 84, 100, CBS_DROPDOWN | WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "", -1, 7, 140, 230, 80, WS_VISIBLE
-    LTEXT "Notația orei:\n\n\
-h = oră     m = minut    s = secundă\nt = am sau pm\n\
-h = 12 ore    H = 24 ore\n\
-hh, mm, ss = cu zerouri   h, m, s = fără zerouri", -1, 13, 150, 214, 50, WS_CHILD | WS_VISIBLE | WS_GROUP
+    LTEXT "Notarea formatelor de oră\n\n\
+h = oră     m = minut     s = secundă     t = am sau pm\n\n\
```
Comment (JoachimHenze) [inline dll/cpl/intl/lang/ro-RO.rc:140]:
```suggestion
h = oră     m = minut     s = secundă     t = am sau pm\n\
```

**Review by GeoB99 — APPROVED** [review]
The translation changes look OK, apart from Joachim's review comments.

File: dll/cpl/intl/lang/ro-RO.rc (hunk 3 above)
Comment (AndreiMiloiu) [reply dll/cpl/intl/lang/ro-RO.rc:33]: I verified. You are right. I will revert this change.

Comment (AndreiMiloiu) [reply dll/cpl/intl/lang/ro-RO.rc]: Fixed!

File: dll/cpl/intl/lang/ro-RO.rc (hunk 4 above)
Comment (AndreiMiloiu) [reply dll/cpl/intl/lang/ro-RO.rc:60]: This is what MS did in XP/2k3. Here is the proof.

File: dll/cpl/intl/lang/ro-RO.rc (hunk 5 above)
Comment (AndreiMiloiu) [reply dll/cpl/intl/lang/ro-RO.rc:63]: Yes. MS did that here. Here is the proof:

2024-08-17 00_46_04-Resource Hacker - intl cpl mui

**Comment by AndreiMiloiu** [comment]
Is it good now?

**Pull request merged**